src/cloudai/_core/test.py

```diff
@@ -41,8 +41,6 @@ class Test:
         sol (Optional[float]): Speed-of-light performance for reference.
```

Comment:
We should remove num_nodes and nodes from the docstring as well.

Reply:
Done.